### PR TITLE
added e2e test case for edit content

### PR DESCRIPTION
**Pull request by NileshPatel17**
added e2e test for #916

### CODE REVIEW

File: apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        cy.get('[data-test="content-context-menu-edit"]').click()
+        cy.get('#DisplayName').type(' Test')
+        cy.contains('Submit').click()
+        cy.get(`[data-test="table-cell-IT Workspace Test"]`).should('have.text', 'IT Workspace Test')
+        //  breadcrumb test
+        const expectedBreadcrumbItems = ['Content', '/', 'IT Workspace Test']
+        cy.get('[data-test="Content"]').click()
+        cy.get('[data-test="menu-item-IT Workspace Test"]').click()
+        cy.get('nav[aria-label="breadcrumb"] li').each(($el) => {
+          console.log($el.text())
```
Comment (herflis) [inline apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts:22]: remove the console log please

File: apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+describe('Edit Content', () => {
+  before(() => {
+    cy.login()
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+  })
+  it('Test case 1: edit content should works properly.', () => {
```
Comment (herflis) [inline apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts:8]:
```suggestion
  it('Test case 1: edit content should work properly.', () => {
```

File: apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        const expectedBreadcrumbItems = ['Content', '/', 'IT Workspace Test']
+        cy.get('[data-test="Content"]').click()
+        cy.get('[data-test="menu-item-IT Workspace Test"]').click()
+        cy.get('nav[aria-label="breadcrumb"] li').each(($el) => {
+          console.log($el.text())
+          expect(expectedBreadcrumbItems).to.include($el.text())
+        })
+      })
+  })
+  it('Test case 2: edit content should works properly.', () => {
```
Comment (herflis) [inline apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts:27]:
```suggestion
  it('Test case 2: edit content should work properly.', () => {
```

**Review by herflis — CHANGES_REQUESTED** [review]

File: apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts (hunk 1 above)
Comment (NileshPatel17) [reply apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts]: Removed

File: apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts (hunk 2 above)
Comment (NileshPatel17) [reply apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts]: Done

File: apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts (hunk 3 above)
Comment (NileshPatel17) [reply apps/sensenet/cypress/integration/edit-content/edit-content.spec.ts]: Done

**Review by herflis — CHANGES_REQUESTED** [review]

**Comment by NileshPatel17** [comment]
@herflis Changes are done.

**Comment by herflis** [comment]
Thank you @NileshPatel17

**Pull request merged**